serverless/serverless-components
# Add faker REST API example

## What has been implemented?

Add an example REST API to generate and return fake data.

Closes SC-174

## Steps to verify

* Run `deploy` to deploy resources
* See `README.md` for usage
* Run `remove` to remove deployed resources

## Todos:
